internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -1782,6 +1782,10 @@ private ConfigKeys() {
   public static final String UNREGISTER_METRIC_FOR_DELETED_STORE_ENABLED =
       "unregister.metric.for.deleted.store.enabled";
 
+  /**
+   * Config to enable single leader replica disabling.
+   */
+  public static final String DISABLE_ERROR_LEADER_REPLICA_ENABLED = "disable.error.leader.replica.enabled";
```

Comment:
The config name is a little confusing, and maybe we can use something like:
`force.leader.replica.fail.over.on.error.enabled`

Reply:
sure